src/zsock.c

```diff
@@ -2202,6 +2202,104 @@ zsock_test (bool verbose)
     zframe_destroy (&frame);
     zmsg_destroy (&msg);
 
+#ifdef ZMQ_STREAM
+    zsock_t *streamrecv = zsock_new_stream("@tcp://127.0.0.1:1234");
```

Comment:
please do not hard-code the port, otherwise the test will randomly fail - use `*` and retrieve it

Reply:
done